torch/utils/dlpack.py

```diff
@@ -2,3 +2,26 @@
 
 from torch._C import _from_dlpack as from_dlpack
 from torch._C import _to_dlpack as to_dlpack
+
+torch._C._add_docstr(from_dlpack, r"""from_dlpack(dlpack) -> Tensor
+
+Decodes a DLPack to a tensor.
+
+Arguments::
```

Comment:
This renders incorrectly (see https://pytorch.org/docs/master/dlpack.html vs. other docs).

Can you change it to:
```
Args:
  dlpack: a PyCapsule object with the dltensor
```
Note the colon in the second line. Thanks!

Reply:
Indeed. Thanks! I'll fix that with my next doc PR.